mintapi/api.py

```diff
@@ -339,7 +342,7 @@ def _sign_in(email, password, driver, mfa_method=None, mfa_token=None,
         except (NoSuchElementException, StaleElementReferenceException, ElementNotVisibleException):
             pass
 
-        driver.implicitly_wait(1)  # seconds
```

Comment:
Cool set of changes, I didn't even know this was a thing! What is the thought behind removing this wait?

Reply:
i'm pretty sure driver.implicitly_wait() does not perform a wait itself, but is just a setting on the driver for how long it will waitFor an element to be found in the DOM before throwing a NotFound exception.  setting this to 1 just says that all subsequent find element calls on driver will try for up to 1s before throwing NotFound.

i'm not a python guy so bit out of my element here... just basing this off my experiences doing this in java/groovy with selenium.